lib/_debugger.js

```diff
@@ -159,7 +159,7 @@ function Client() {
     protocol.execute(d);
   });
 
-  protocol.onResponse = this._onResponse.bind(this);
```

Comment:
This one takes one argument, `res` (response).

If you search the file for `_onResponse` (The function being bind), you should be able to tell what arguments it can take: https://github.com/nodejs/node/blob/master/lib/_debugger.js

So we need to update the change to look like `protocol.onResponse = (res) => this._onResponse(res);`

(Note: if anything uses `arguments`, we need to look at in more detail.)